proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,452 @@
+   maintainers should happen in a timely manner, even (especially!) if that’s
+   saying no. An initial response to open PRs should occur within [TBD] business
+   days.
+1. A contribution should not be left abandoned. The maintainers are responsible
+   see it through to a conclusion.
+1. Contributors should know exactly what to expect when interacting with the
+   maintainers. The current status should be clearly indicated and the next
+   steps well defined.
+
+## Specification
+
+ ![Figure A: Flowchart of contribution guide review
+process.](contrib-guide-high-level.png "Figure A: Flowchart of contribution
+guide review process.")
+
+**Figure A: Flowchart of contribution guide review process.**
+
+To improve the review process for feature requests and pull requests, we propose
+to modify the existing stages and expand them. Figure A demonstrates how a
+request and contribution will proceed through this improved process. The process
+extends and clarifies the three existing stages: 
+
+1. Intake, where feature requests are triaged and those suitable for community
+   contribution are identified;
+2. Implementation, where a contributor acknowledges that they will work on an
+   issue and open a pull request, and
+3. Review, where pull requests are prioritized for review and the maintainers
+   work with the contributor to resolve the contribution.
+
+In addition, we propose to publish an improved contributing guide to document
+the review process, acceptance criteria, and expectations for communication. In
+addition, a transparent status must be published to indicate what is available
+for contribution and how close a contribution is to completion.
+### Intake
+
+The intake process identifies feature requests, through GitHub issues, where
+community contributions are welcome. The maintainers will actively review GitHub
+issues for feature requests to make this determination. Depending on the issue
+type (bug or feature request) and area (code or documentation), a set of
+criteria must be met before marking it as ready for contribution. The criteria
+used to make that decision may include:
+
+1. The change is feasible for a community member to contribute. See the appendix
+   for a definition of what would not be feasible for community contribution.
+1. There is enough detail provided for the maintainers to agree with the
+   implementation plan.
+
+Specifically for feature requests, the change must impact a minimum of 10 of
+users, measured by GitHub reactions.
+
+Once these criteria have been met, an issue is available for the community to
+contribute. The issue will be labeled and added to a queue where users
+interested in contributing can discover what is available.
+
+All issues are subject to existing automated issue closure mechanisms, which
+will prompt for community input before they are automatically closed if there
+are not enough upvotes ([TBD]) and no activity (comments or upvotes) for [TBD;
+currently a year].
```

Comment:
> whether "contribution-ready" issues are subjected to the auto-closure

By the current auto-closure mechanism they actually are not (since it's `> 10 upvotes OR activity in the last year`). I'm going to move it to the "Implementation" section for now, since it might explain how something transitions off of the GitHub project.

Reply:
I think its important that if issues get stale AND they do not meet the impact threshold, they will automatically be closed.

Reply:
Left it here and clarified that it's for issues that do not meet the criteria.